modules/data/pdf_viewer.components.json

```diff
@@ -1 +1,13 @@
-{}
\ No newline at end of file
+{
```

Comment:
Is the pdf viewer part of the chrome context? If so, you should add `"context": "chrome"`. If not, this should probably be part of `Generic_Pdf` (potentially with a rename).

Reply:
fixed